lib/kernel/doc/guides/socket_usage.md

```diff
@@ -524,3 +524,69 @@ Options for level `sctp`:
 | rtoinfo           | sctp_rtoinfo()         | yes | yes | none                            |
 
 _Table: sctp options_
+
+
+[](){: #socket_configure_flags }
```

Comment:
```suggestion
```

There is no need to have this, you can link directly to the header: `socket_usage.md#socket-configure-flags`.